
package ru.yandex.practicum.dto;

import java.util.ArrayList;
import java.util.List;

public class WarehouseCheckRequestDto {
    private List<CartItemDto> items;

    public List<CartItemDto> getItems() {
        return items;
    }

    public void setItems(List<CartItemDto> items) {
        this.items = items;
    }

    public void addItem(Long productId, int quantity) {
        if (items == null) items = new ArrayList<>();
        items.add(new CartItemDto(productId, quantity));
    }
}
